### ISSUE TITLE
Volume / Shuffle / Repeat Buttons not working

### ISSUE THREAD

**Issue by Lausbuab1** [Rello/audioplayer#128]
### Expected behavior
That I can use the buttons on the music player on the top right side.

### Actual behavior
Can't use any button, I can click them, but nothing happens

### Steps to reproduce the behavior
1. Login
2. Click on the buttons
3. Nothing is happening

### Server configuration
**Operating system**:
Debian 8

**Web server:**
Apache

**Database:**

**PHP version:**

**ownCloud/Nextcloud version:** (see `/status.php`)
11.0.1

**Updated from an older ownCloud/Nextcloud or fresh install:**
9.X to 10.X and then to 11.X (Installed Audio Player first time on 11.0.1)

**Audio Player version:** (see `/settings/apps`)
1.4.0

**Where did you install Audio Player from:**
Built-in AppStore

**Are you using external storage, if yes which one:** `local/smb/sftp/...`
No

**Are you using encryption:** yes/no
Yes

### Client configuration
**Operating system:**
Windows 10

**Browser:**
Internet Explorer, Chrome and Firefox

#### ownCloud/Nextcloud log
<details>
<summary>ownCloud/Nextcloud log (`/data/[owncloud|nextcloud].log`)</summary>
```
Insert your ownCloud/Nextcloud log here
```
</details>

#### Web server error log (optional)
<details>
<summary>Web server error log</summary>
```
Insert your webserver log here
```
</details>

#### Browser log (optional)
<details>
<summary>Browser log</summary>
```
Insert your browser log here
```
</details>

**Comment by Rello** [comment]
hello,
what exactly is the issue? just just checked on OC and NC and it works fine for me.

when you select e.g. shuffle, it will become black.
when you play a song and press the forward button, it will jump to another shuffled song.
same for repeat. when it is enabled, the playlist will restart after reaching the end.

let me know what does not work on your side

**Comment by Lausbuab1** [comment]
Hi,

after I've tested more things, I've found out, that I have the same issue, that occurs, if the left side bar is actibe, that the other bottons on the top left don't work. 

Issue is also opened on your github and you said, you will fix this on 1.41.

**Comment by ghost** [comment]
@Lausbuab1 Issue #111 ?

**Comment by Lausbuab1** [comment]
Yes, this issue. 

Have this problem not only in chrome, IE and firefox also.

**Comment by ghost** [comment]
> Internet Explorer, Chrome and Firefox

Internet Explorer and Edge are not supported. ;-)
https://github.com/rello/audioplayer/wiki/installation-instructions#known-issues

**Comment by Lausbuab1** [comment]
Then it's just firefox and chrome :-)

PS: Great work, really appreciate it :)

**Comment by Rello** [comment]
@Lausbuab1 nevertheless, we are searching CSS support for #111 because we could not solve it yet.
so if you have skills here... :-)

**Comment by Lausbuab1** [comment]
Sorry, I'm a noob in CSS :-(

**Comment by RobotFishBox** [comment]
I'm running this on NextCloud 12.0.4 and in chrome. The volume button is the only button that seems to be giving me trouble. However, after I press it I can still control the volume with the arrow keys. Anybody else have the same behavior?

**Comment by Rello** [comment]
Hello,
the volume button is also connected to this sidebar-offset issue.
when the sidebar is closed, its working as expected

**Comment by Rello** [comment]
Issue was located; Testing in progress

**Comment by Rello** [comment]
Released with 2.2.2
